tests/regression/mmrotate.yml

```diff
@@ -48,3 +48,21 @@ models:
       - *pipeline_ort_detection_dynamic_fp32
       - *pipeline_trt_detection_dynamic_fp32
       - *pipeline_trt_detection_dynamic_fp16
+
+  - name: oriented_rcnn
+    metafile: configs/oriented_rcnn/metafile.yml
+    model_configs:
+      - configs/oriented_rcnn/oriented_rcnn_r50_fpn_fp16_1x_dota_le90.py
+    pipelines:
+      - *pipeline_ort_detection_dynamic_fp32
+      - *pipeline_trt_detection_dynamic_fp32
+      - *pipeline_trt_detection_dynamic_fp16
+
+  - name: gliding_vertex
+    metafile: configs/gliding_vertex/metafile.yml
+    model_configs:
+      - configs/gliding_vertex/gliding_vertex_r50_fpn_1x_dota_le90.py
+    pipelines:
+      - *pipeline_ort_detection_dynamic_fp32
```

Comment:
If `onnxruntime` is supported, we'd better update tables in `benchmark` and `suporrt_models` to `Y`